include/mbedtls/armv8ce_aes.h

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
Honestly, AES-NI was just doing it the same way as Via Padlock support, so the person who knows the reasons if Paul.

However, if I hazard a guess as to why, I think it's mainly because it looked convenient (avoid clutter in `aes.c`) and concerns about minimizing the exposed API and ABI were probably given less importance at that time than now.